tests/SprykerEcoTest/Yves/Computop/Mapper/Init/PrePlace/PayPalExpressToQuoteMapperTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Yves\Computop\Mapper\Init\PrePlace;
+
+use Codeception\TestCase\Test;
+use SprykerEco\Yves\Computop\Mapper\Init\PrePlace\PayPalExpressToQuoteMapper;
+use SprykerEcoTest\Yves\Computop\Mapper\PayPalExpressToQuoteMapperTestConstants;
+
+class PayPalExpressToQuoteMapperTest extends Test
+{
+    /**
+     * @var \SprykerEcoTest\Yves\Computop\ComputopYvesTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testMapAddressTransfer(): void
+    {
+        //Arrange
+        $mapper = $this->createPayPalExpressToQuoteMapper();
+
+        //Act
+        $quoteTransfer = $mapper->mapAddressFromComputopPayPalExpressInitResponseToQuote(
+            $this->tester->createComputopPayPalExpressInitResponseTransfer(),
+            $this->tester->createQuoteTransfer()
+        );
+
+        //Assert
+        $this->assertNotNull($quoteTransfer->getShippingAddress());
+        $this->assertNotNull($quoteTransfer->getShippingAddress()->getCountry());
+        $this->assertEquals(
+            PayPalExpressToQuoteMapperTestConstants::ADDRESS_COUNTRY_CODE_VALUE,
+            $quoteTransfer->getShippingAddress()->getCountry()->getIso2Code()
+        );
+        $this->assertEquals(
+            PayPalExpressToQuoteMapperTestConstants::FIRST_NAME_VALUE,
+            $quoteTransfer->getShippingAddress()->getFirstName()
+        );
+
+        foreach ($quoteTransfer->getItems() as $itemTransfer) {
+            $this->assertEquals(
```

Comment:
```suggestion
            $this->assertSame(
```

please use assertSame everywhere when you compare non-object values